packages.json

```
@@ -18012,7 +18012,7 @@
     "license": "MIT",
     "web": "https://github.com/Ethosa/nodesnim"
   },
-  {
+  {   
```

```suggestion
  {
```
only suggesting this for myself :)